examples/pytorch/vgae/preprocess.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+import scipy.sparse as sp
+import torch
+
+
+def mask_test_edges(adj):
```

Comment:
Have we added the option not to add reverse edges in dataset?

Reply:
Not yet. The related pr(https://github.com/dmlc/dgl/pull/2588)  still requests review. Now, if we choose dgl dataset, it will add reverse edges by default, while choosing website dataset won't.

Reply:
Have added option not to add reverse edges. 
The new commit uses dgl dataset graph without reverse edges and has better performance.